src/plugins/SoundPlugin.js

```diff
@@ -305,11 +243,68 @@ export class SoundPlugin extends ButtonPlugin {
   }
 
   /**
-   *
-   *
    * @memberof SoundPlugin
    */
-  opened() {
+  init() {
+    this.client.on(
+      'features',
+      function(features) {
+        if (
```

Comment:
I don't think we should overwrite the display property to `inline-block` because that'll potentially prevent devs from providing their own custom positioning. Instead, I think we should just set it to an empty string, to [allow it to default](https://stackoverflow.com/a/2028029/10200077) to the stylesheet on the page (or the default browser behavior)